content/firstabout.html

```diff
@@ -1,24 +1,24 @@
-<a href="http://www.usfirst.org">FIRST (For Inspiration and Recognition of Science and Technology)</a> is a world-wide organization started
-by Dean Kamen. It was designed to encourage students to pursue careers in science and technology,
-and to instill a sense of Gracious Professionalism. Gracious Professionalism encourages students to work
-together to help other teams when needed and to treat everyone with kindness and respect. Because of
-this, FIRST is said to not only teach students to build robots, but to build character as well.
+<a href="http://www.usfirst.org">FIRST (For Inspiration and Recognition of Science and Technology)</a> is a world-wide organization founded 
+by Dean Kamen in 1992. FISRT was designed to encourage students to pursue S.T.E.M. related careers 
+while instilling a sense of Gracious Professionalism. Gracious Professionalism is the mutual respect of oneself and others within 
+the heat of competition. FIRST is built around four levels of robotics competition. <a href="http://www.usfirst.org/roboticsprograms/jr.fll">Jr.FLL(grades K-3)</a>, 
+<a href="http://www.usfirst.org/roboticsprograms/fll">FLL(4-8)</a>, 
+<a href="http://www.usfirst.org/roboticsprograms/ftc">FTC(7-12)</a>, and 
+<a href="http://www.usfirst.org/roboticsprograms/frc">FRC(9-12)</a> enable students of all ages to 
+engage in FIRST.
```

Comment:
"enable students of all ages to engage in FIRST" seems like the end of a sentence to me but it starts off with Jr.FLL as far as I can tell. Am I missing something?

I also think there should be a space between the name and the grades e.g. "FLL (4-8)".